packages/backend/financial-health-calculator/index.mts

```diff
@@ -0,0 +1,21 @@
+import { calculateFinancialHealthScore } from '@dependency-health-monitor/financial-health-calculator';
+import type { AzureFunction, Context, HttpRequest } from '@azure/functions';
+
+const httpTrigger: AzureFunction = async function (context: Context, req: HttpRequest): Promise<void> {
```

Comment:
Can you add the following line?
`context.res.type('application/json');`

Reply:
Resolved in d8446cb24a86f01371f0dbb722b68ad87f93e311